data.js

```diff
@@ -3579,6 +3579,24 @@ module.exports = [
     twitter: "@wuniverse_site",
     tags: ["PHP", "JavaScript", "CSS", "WordPress"],
   },
+
+  {
+    name: "Vipul Soni",
+    description: "Full Stack Web Developer.Write blogs about Web Developement",
+    url: "https://medium.com/@vipulsoniab17_40156",
+    twitter: "https://twitter.com/Vipulso06279659",
```

Comment:
Pleas fix your twitter handle, it should be `@Vipulso06279659` not the entire URL